<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NatusferaQGIS3DialogBase</class>
 <widget class="QDialog" name="NatusferaQGIS3DialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NatusferaQGIS 3</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="username_line_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load_csv_username_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="project_line_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load_csv_project_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="species_line_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="load_csv_species_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="attention_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Attention&lt;/b&gt;: if you are going to use data from Natusfera, check &lt;br&gt;the
licence of each observation.</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;style type=&quot;text/css&quot;&gt;
    span{
    color: #000000;
    font-size:8pt;
    font-style:italic;
    text-decoration: underline;
    }
&lt;/style&gt;

Username &lt;span&gt;(http://natusfera.gbif.es/observations)&lt;/span&gt;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="project_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;style type=&quot;text/css&quot;&gt;
    span{
    color: #000000;
    font-size:8pt;
    font-style:italic;
    text-decoration: underline;
    }
&lt;/style&gt;

Project &lt;span&gt;(http://natusfera.gbif.es/projects)&lt;/span&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="species_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;style type=&quot;text/css&quot;&gt;
    span{
    color: #000000;
    font-size:8pt;
    font-style:italic;
    text-decoration: underline;
    }
&lt;/style&gt;

Species &lt;span&gt;(http://natusfera.gbif.es/taxa)&lt;/span&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/plugins/natusfera_qgis_3/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_csv_everything_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Load Everything</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
